sentry-android-core/src/main/java/io/sentry/android/core/NetworkBreadcrumbsIntegration.java

```diff
@@ -63,40 +63,71 @@ public void register(final @NotNull IHub hub, final @NotNull SentryOptions optio
         "NetworkBreadcrumbsIntegration enabled: %s",
         androidOptions.isEnableNetworkEventBreadcrumbs());
 
+    this.options = options;
+
     if (androidOptions.isEnableNetworkEventBreadcrumbs()) {
 
       // The specific error is logged in the ConnectivityChecker method
-      if (buildInfoProvider.getSdkInfoVersion() < Build.VERSION_CODES.LOLLIPOP) {
-        networkCallback = null;
-        logger.log(SentryLevel.DEBUG, "NetworkBreadcrumbsIntegration requires Android 5+");
+      if (buildInfoProvider.getSdkInfoVersion() < Build.VERSION_CODES.N) {
```

Comment:
I've changed this to `N`, as our `AndroidConnectionStatusProvider.registerNetworkCallback` itself only supports >= `N`